src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,273 @@
+import { cva, VariantProps } from "class-variance-authority";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const shellStylesOuter = cva(
```

Comment:
I don't think we need any variants for this page template - you can omit cva for now. Just use the primary style you've set up here and drop the secondary one.